doc/quint.md

```diff
@@ -256,6 +256,75 @@ Options:
 
    The errors and warnings are written in the format of [ADR002][].
 
+## Command verify
+
+```sh
+$ quint verify <input>
+
+Verify a Quint specification via Apalache
+
+Options:
+  --help             Show help                                     [boolean]
+  --version          Show version number                           [boolean]
+  --how              how to do verification: bounded model checking (bmc) or
+                     symbolic execution (symexec)
+                   [string] [choices: "symexec", "bmc"] [default: "symexec"]
+  --main             name of the main module (by default, computed from
+                     filename)                                      [string]
+  --out              output file (suppresses all console output)    [string]
+  --out-itf          output the trace in the Informal Trace Format to file
+                     (supresses all console output)                 [string]
+  --max-samples      the number of symbolic paths to try (when using
+                     --how=symexec)                [number] [default: 10000]
+  --max-steps        the maximum on the number of steps in every trace
+                                                      [number] [default: 10]
+  --init             name of the initializer action
+                                                  [string] [default: "init"]
+  --step             name of the step action      [string] [default: "step"]
+  --invariant        invariant to check: a definition name or an expression
+                                                [string] [default: ["true"]]
+  --seed             random seed to use for symbolic execution (when using
+                     --how=symexec)                                 [string]
+  --apalache-config  Filename of the additional Apalache flags in the JSON
+                     format                                         [string]
```

Comment:
```suggestion
  --apalache-config  Filename of the additional Apalache configuration in the HOCON
                     format (a superset of JSON)                           [string]
```

Reply:
Fix in cli.ts, since this help is autogenerated